src/js/tech/html5.js

```diff
@@ -78,25 +77,39 @@ class Html5 extends Tech {
       }
     }
 
+    // TODO: add text tracks into this list
     const trackTypes = ['audio', 'video'];
 
-    // ProxyNativeTextTracks
+    // ProxyNative Video/Audio Track
     trackTypes.forEach((type) => {
+      const elTracks = this.el()[`${type}Tracks`];
+      const techTracks = this[`${type}Tracks`]();
       const capitalType = toTitleCase(type);
 
-      if (!this[`featuresNative${capitalType}Tracks`]) {
+      if (!this[`featuresNative${capitalType}Tracks`] ||
+          !elTracks ||
+          !elTracks.addEventListener) {
         return;
       }
-      const tl = this.el()[`${type}Tracks`];
 
-      if (tl && tl.addEventListener) {
-        tl.addEventListener('change', Fn.bind(this, this[`handle${capitalType}TrackChange_`]));
-        tl.addEventListener('addtrack', Fn.bind(this, this[`handle${capitalType}TrackAdd_`]));
-        tl.addEventListener('removetrack', Fn.bind(this, this[`handle${capitalType}TrackRemove_`]));
+      this[`handle${capitalType}TrackChange_`] = (e) => {
```

Comment:
These add/remove/change method names are repeated below. As long as we're being byte-conscious and DRY, maybe they ought to be defined as variables?